core/src/main/java/com/megaease/easeagent/core/plugin/CommonInlineAdvice.java

```diff
@@ -19,9 +19,9 @@
 
 import com.megaease.easeagent.core.plugin.annotation.Index;
 import com.megaease.easeagent.core.plugin.transformer.advice.AgentAdvice.NoExceptionHandler;
-import com.megaease.easeagent.plugin.interceptor.MethodInfo;
 import com.megaease.easeagent.plugin.api.InitializeContext;
 import com.megaease.easeagent.plugin.bridge.EaseAgent;
+import com.megaease.easeagent.plugin.interceptor.MethodInfo;
```

Comment:
If you don't modify this class, there is no need to make redundant changes

Reply:
ok, i have reverted this line of code.